x509-rest-client/src/main/java/com/c8y/x509/X509RestClient.java

```diff
@@ -66,6 +67,16 @@ private static HttpRequest buildRequest() {
 		}
 	}
 
+	private static HttpRequest buildRequestWhenImmediateIssuerIsPresentInPlatform() {
```

Comment:
did you intentionally make a new method that is not used? Maybe modify an existing method with condition
if (StringUtils.isNotEmpty(LOCAL_DEVICE_CHAIN))
{ then setup header} ??

Reply:
Done